src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  function getPartnerSex(personSex) {
+    return personSex === 'f' ? 'husband' : 'wife';
+  }
+
+  const marriedStatus = isMarried
+    ? `${partnerName} is my ${getPartnerSex(sex)}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      {isMarried ? (
```

Comment:
I believe conditional operator here is redundant as you do it before in marriedStatus, so here you can use p tag only once

Reply:
Good point)